src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -331,14 +331,19 @@ public void fireFileStarted(String fileName) {
     @Override
     public void fireErrors(String fileName, SortedSet<LocalizedMessage> errors) {
         final String stripped = CommonUtils.relativizeAndNormalizePath(basedir, fileName);
+        boolean removeCache = false;
```

Comment:
please rename one more time, I read it before I changed name to more descriptive.